next.config.js

```diff
@@ -15,10 +15,11 @@ module.exports = withPlugins([[indexSearch], [feed], [sitemap], [socialImages]],
 
   // By enabling verbose logging, it will provide additional output details for
   // diagnostic purposes. By default is set to false.
-  //verbose: true,
+  // verbose: true,
 
   env: {
     WORDPRESS_HOST: removeLastTrailingSlash(process.env.WORDPRESS_HOST),
     WORDPRESS_GRAPHQL_ENDPOINT: removeLastTrailingSlash(process.env.WORDPRESS_GRAPHQL_ENDPOINT),
+    POSTS_PER_PAGE: 10,
```

Comment:
can we include this as a "default" inside of the code and allow someone to configure this as optional? that way if someone doesn't mind 10/post they dont have to worry about it. worth adding somethign to the README about too in that case

Reply:
great point. I'll leave that commented and create a function inside the lib like this:

```
export function getPostsPerPage() {
  return Number(process.env.POSTS_PER_PAGE) || 10;
}
```